src/main/plugins/shortcuts-search-plugin/shortcuts-search-plugin.ts

```diff
@@ -22,19 +23,23 @@ export class ShortcutsSearchPlugin implements SearchPlugin, OpenLocationPlugin {
     private readonly filePathExecutor: (filePath: string, privileged: boolean) => Promise<void>;
     private readonly filePathLocationExecutor: (filePath: string) => Promise<void>;
     private readonly commandlineExtecutor: (command: string) => Promise<void>;
```

Comment:
```suggestion
    private readonly commandlineExecutor: (command: string) => Promise<void>;
```

You have a typo here

Reply:
I saw this misspell and I didn't want to change it tbh since it was oliver's code.

Reply:
I guess no harm in fixing that tho. I will add a commit to fix it.

Reply:
This was oliver's code, I didn't want to change it but i guess there is no harm in fixing a misspell.